package.json

```diff
@@ -26,7 +26,7 @@
     "styled-components": "^5.0.0 || ^6.0.0"
   },
   "dependencies": {
```

Comment:
NOTE: I can't comment on the `peerDependencies`, but before we merge we'll want to make sure:
- grommet is at latest version
- grommet-icons has been replaced with hpe-icons ^1.0.0